address_translators.go

```diff
@@ -0,0 +1,24 @@
+package gocql
+
+// AddressTranslator provides a way to translate node addresses (and ports) that are
+// discovered or received as a node event. This is especially useful in ec2 (when
+// using the EC2MultiRegionAddressTranslator) to translate public IPs to private IPs
+// when possible.
+type AddressTranslator interface {
```

Comment:
Im still considering if this function should return an error, though I guess we should leave that for the HostFilter?